### DISCUSSION TITLE
Best way to view recordings

### DISCUSSION THREAD

**Discussion by pbix** [blakeblackshear/frigate#3560] [Ask A Question]
I have been using Frigate for over a year now and I must say it is very reliable and effective.  I have about 6 cameras around the house all streaming everything to disk.  Very cool project that I would recommend to anyone.

But the recordings show up as 10 second snippets on my hard drive which makes it pretty hard to search through to say see the rabbit that got into the garden.

What is the best way to load all these video files so that I an use a slider to quickly scan through a whole day of recordings when I am searching for something but don't know what time it was in the video.

**Comment by NickM-27** [comment]
The frontend is best or you can use the API to pull a stream from a given start and end time 

https://docs.frigate.video/integrations/api#get-vodcamerastartstart-timestampendend-timestampindexm3u8

**Reply by pbix** [reply]
I looked at the API documentation referecne and there are no examples given.  Could you link me to a reference to show me how to do this exactly.

What would be really neat is a view with slide to allow me to view all recorded video simultaneously and a slide bar to go forward and backwards in time.  Right now its pretty tedious to find something that happened in the past.

**Reply by NickM-27** [reply]
The API I linked is exactly what is used to get a steam or there's one to download an MP4 for a specified start and end time. The UI also has the recordings view where you can view one hour at a time and easy move between them

**Comment by ryancurrah** [comment]
Has this changed at all? I don't see a good way to do this today.

**Reply by NickM-27** [reply]
There are many different ways:

1. Open a camera and choose history
2. Open review and choose a review item
3. Open review, go to motion tab, scrub until time of interest then open camera

**Reply by ryancurrah** [reply]
Awesome I did not know about the History area. Thank you!
